html/semantics/embedded-content/the-iframe-element/iframe_javascript_url_in_src.htm

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<title>javascript: URL in iframe src</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#process-the-iframe-attributes">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+const tests = [
+  // desc, url, expected body.textContent
+  ['string', 'javascript:"foo"', 'foo'],
+  ['empty string', 'javascript:""', ''],
+  ['String object', 'javascript:new String("foo")', ''],
+  ['undefined', 'javascript:void(0)', ''],
+  ['number', 'javascript:1', ''],
+  ['boolean', 'javascript:true', ''],
+  ['null', 'javascript:null', ''],
+  ['global', 'javascript:window', ''],
+  ['host object', 'javascript:document', ''],
+  ['function', 'javascript:(() => { return function() {}; })()', ''],
+  ['regexp', 'javascript:/foo/', ''],
+  ['array', 'javascript:["foo"]', ''],
+  ['object', 'javascript:{"foo": "bar"}', ''],
+  ['ArrayBuffer', 'javascript:new ArrayBuffer(8)', ''],
+  ['TypeError', 'javascript:new TypeError("foo")', ''],
+];
+for (const [desc, url, expected_textContent] of tests) {
+  async_test(t => {
+    const iframe = document.createElement('iframe');
+    iframe.src = url;
+    iframe.hidden = true;
+    let load_events = 0;
+    iframe.onload = t.step_func(() => {
+      load_events++;
```

Comment:
Add an assert before this that load_events is 0? And maybe make it a boolean? That way we can exit early and not wait 100ms.

Reply:
I considered that but I figured out would be nice to know how many `load` events you got if there are more than 1. All browsers currently pass at least 1 subtest (which take 100ms), so the test as a whole doesn't complete faster if some subtests get too many `load` events. The subtests also run in parallel.